stan/math/rev/fun/variance.hpp

```diff
@@ -38,12 +38,14 @@ inline var calc_variance(size_t size, const var* dtrs) {
  * @param[in] v a vector
  * @return sample variance of specified vector
  */
-inline var variance(const std::vector<var>& v) {
+template <typename StdVec, require_std_vector_t<StdVec>* = nullptr,
+          require_vt_var<StdVec>* = nullptr>
+inline auto variance(StdVec&& v) {
```

Comment:
why auto return type?

Reply:
I like using auto but I'll switch these to var